src/py42/modules/securitydata.py

```diff
@@ -202,32 +202,31 @@ def _find_file_versions(self, md5_hash, sha256_hash):
                 # 'get_file_location_detail_by_sha256', hence we keep looking until we find a stream
                 # to return
                 debug.logger.warning(
-                    u"PDS fetch file version API failed for md5 hash {0} and sha256 hash {1}. "
+                    u"Failed to find any file version for md5 hash {0} / sha256 hash {1}. "
                     u"Error: ".format(md5_hash, sha256_hash),
                     err,
                 )
-                pass
 
     def _stream_file(self, file_generator, checksum):
         for response in file_generator:
+            url = response[u"storageNodeURL"]
```

Comment:
This breaks if the response data isn't a dict, like a `<Py42Response [status=204, data='']>

Reply:
good find, thanks